pkg/koordlet/metricsadvisor/collectors/beresource/be_resource_collector.go

```diff
@@ -88,31 +87,42 @@ func (b *beResourceCollector) collectBECPUResourceMetric() {
 		return
 	}
 
-	beCPURequest := b.getBECPURequestSum()
+	beCPUMilliRequest := b.getBECPURequestMilliSum()
 
-	beCPUUsageCores, err := b.getBECPUUsageCores()
+	beCPUUsageMilliCores, err := b.getBECPUUsageMilliCores()
 	if err != nil {
 		klog.Errorf("getBECPUUsageCores failed, error: %v", err)
 		return
 	}
 
-	if beCPUUsageCores == nil {
-		klog.Info("beCPUUsageCores is nil")
+	collectTime := time.Now()
+	beLimit, err01 := metriccache.NodeBEMetric.GenerateSample(
+		metriccache.MetricPropertiesFunc.NodeBE(string(metriccache.BEResourceCPU), string(metriccache.BEResouceAllocationRealLimit)), collectTime, float64(realMilliLimit))
+	beRequest, err02 := metriccache.NodeBEMetric.GenerateSample(
+		metriccache.MetricPropertiesFunc.NodeBE(string(metriccache.BEResourceCPU), string(metriccache.BEResouceAllocationRequest)), collectTime, float64(beCPUMilliRequest))
+
+	beUsage, err03 := metriccache.NodeBEMetric.GenerateSample(
+		metriccache.MetricPropertiesFunc.NodeBE(string(metriccache.BEResourceCPU), string(metriccache.BEResouceAllocationUsage)), collectTime, float64(beCPUUsageMilliCores))
+
+	if err01 != nil || err02 != nil || err03 != nil {
+		klog.Errorf("failed to collect node BECPU, err01: %v, err02: %v, err03: %v", err01, err02, err03)
```

Comment:
Please revise the variable naming and logging for readability and self-defined.
`err01` is not a recommended naming, and the current log message is only understandable to the developers who have the codebase.
Same below.